components/google-cloud/dependencies.py

```diff
@@ -0,0 +1,34 @@
+# Lint as: python3
+# Copyright 2020 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Setup configuration of  Google Cloud Components client side libraries."""
+
+
+def make_required_install_packages():
+  return [
```

Comment:
Shall we consider making the dependencies version-bound?

Reply:
+1.

Reply:
Done added version for kfp and aiplatfrom dependancies.